lib/semantics/mod-file.cc

```diff
@@ -55,11 +66,11 @@ static std::ostream &PutAttrs(std::ostream &, Attrs,
 static std::ostream &PutAttr(std::ostream &, Attr);
 static std::ostream &PutType(std::ostream &, const DeclTypeSpec &);
 static std::ostream &PutLower(std::ostream &, const std::string &);
-static bool WriteFile(const std::string &, std::string &&);
+static int WriteFile(const std::string &, const std::string &);
```

Comment:
What does the `int` result signify?  errno?

Reply:
Yes. I've updated the comment on the function definition.